docs/rancher/rke2-cluster.md

```diff
@@ -42,3 +42,21 @@ Users can create a RKE2 Kubernetes cluster from the **Cluster Management** page
     RKE2 v1.21.5+rke2r2 or above provides a built-in Harvester Cloud Provider and Guest CSI driver integration.
 
     - Currently only imported Harvester clusters are supported automatically.
+
+
+### Using Harvester RKE2 Node Driver in Air Gapped Environment
+
+RKE2 provisioning relies on the qemu-guest-agent to get the IP of the virtual machine.
+It may not be feasible to install qemu-guest-agent in an air gapped environment.
```

Comment:
```suggestion
RKE2 provisioning relies on the `qemu-guest-agent` to get the IP of the virtual machine. However, it may not be feasible to install `qemu-guest-agent` in an air gapped environment.
```